### PR TITLE
Fix the issue with context menu and multiple groups

### PR DESCRIPTION
**Pull request by ramramps**
This is the fix for two issues

MAGN-6924 Create Group should not be first on the context menu
MAGN-6925 Pressing Ctrl-G twice darkens the default pink color for a group
MAGN-6926 Default group title text should be the same size as node titles and get only larger with font increase.
MAGN-6927 Notes don't respect group equal group margin rule.
MAGN-6928 changing note size doesn't update group until group is tweaked
MAGN-6929 Group title should be able to be deleted
MAGN-6930 Ctrl-G doesn't work on notes

**Fix - MAGN-6924** 
 1. Updated the context menu on Nodes / Notes.

**Fix - MAGN-6925** 
   Checked the condition on workspacemodel when creating a group. Ideally there should not be any groups selected when creating a new group. This avoids creating many groups at the same location (or say same nodes / notes). 

**Fix - MAGN-6926** 
1. Changed the default font size to 14.

**Fix - MAGN-6927,**Fix - MAGN-6928** **  
 Updated the boundary selection on Text Changed event (Notes);

**Fix - MAGN-6929**
  Removed the empty string check on the converter.This will allow empty annotation text.

**Fix - MAGN-6930**
 Included Notes on Ctrl G

- [x] @pboyer   - Few more minor fix'es.

### CODE REVIEW

File: src/DynamoCore/Models/WorkspaceModel.cs [hunk 1]
```diff
@@ -668,19 +668,25 @@ public AnnotationModel AddAnnotation(string text, Guid id)
         {
             var selectedNodes = this.Nodes == null ? null:this.Nodes.Where(s => s.IsSelected);
             var selectedNotes = this.Notes == null ? null: this.Notes.Where(s => s.IsSelected);
-
-            var annotationModel = new AnnotationModel(selectedNodes, selectedNotes)
+            //No Groups should be selected when creating a new group. This is to avoid creating multiple groups
+            //on the selected nodes.
+            var selectedAnnotation = this.Annotations == null ? null : this.Annotations.Where(s => s.IsSelected);
+            if (selectedAnnotation != null && !selectedAnnotation.Any())
```
Comment (pboyer) [inline src/DynamoCore/Models/WorkspaceModel.cs]: I think this is fine for now, but wouldn't it make more sense to deselect the annotation and do the grouping?

**Comment by pboyer** [comment]
LGTM.

File: src/DynamoCore/Models/WorkspaceModel.cs (hunk 1 above)
Comment (ramramps) [inline src/DynamoCore/Models/WorkspaceModel.cs]: Sure @pboyer - I am checking on that behavior as well. There is a different task to check for empty groups and that might handle this I guess. I will check on that.

**Comment by pboyer** [comment]
LGTM, part deux.

**Pull request merged**